### ISSUE TITLE
Recipe for orange brick stairs incorrect

### ISSUE THREAD

**Issue by Not-February** [maideniles/Maiden-s-Marvelous-Materials#2]
I found another overlooked recipe; this time it's the orange brick stairs recipe. Currently it is trying to use vanilla stone bricks instead of the orange brick block.

**Comment by maideniles** [comment]
i'll work on fixing that, will have the fix up asap. thanks :)

**Comment by maideniles** [comment]
so i'm having a problem with eclipse using old versions of the files when it compiles the .jar.  I checked the source, and the recipe in the actual source folder here is already correct, but every time eclipse compiles the .jar, it has that old broken recipe that I fixed ages ago.  This is actually the reason for all of the corrections I've had to make since I released 2.0... I even showed a friend what it was doing when it started. The source files were fine but the compiled ones were broken... I've refreshed, reloaded, restarted eclipse, and it's still doing it...I'm going to have to figure something out, because this refuses to work correctly. I will keep you posted...I will probably end up uninstalling and reinstalling eclipse. it's the only thing I've not done yet.
